frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -582,9 +614,26 @@ pub mod pallet {
 		/// Duration of the unsigned phase.
 		#[pallet::constant]
 		type UnsignedPhase: Get<Self::BlockNumber>;
+
 		/// Duration of the signed phase.
 		#[pallet::constant]
-		type SignedPhase: Get<Self::BlockNumber>;
+		type SignedPhase: Get<Self::BlockNumber>; // TODO(gpestana): replace by `CommitReveal`
+
+		/// Duration of the commit reveal phase.
+		#[pallet::constant]
+		type CommitReveal: Get<Self::BlockNumber>;
```

Comment:
I know this was how the phase durations were named before. But I feel it would be better if we name these associated types as `CommitRevealDuration`, `CommitPhaseDuration` and so on. Wdyt?